### ISSUE TITLE
web admin: Services menu reports wrong state for some services

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#4817]
Some services are marked as "running" but in reality they are disabled (PID=0). Consequently, a warning is displayed to user (next to service name)

**Screenshots**
![Capture d’écran de 2019-09-17 16-52-13](https://user-images.githubusercontent.com/10830260/65053167-d0d72700-d96b-11e9-9267-3fa987bf1681.png)

**Expected behavior**
Stopped state for services not running.

**Additional context**
Debian 9.

**Comment by nqb** [comment]
API response for `fingerbank-collector`, a service that is not running on my setup:
```json
{"alive":1,"enabled":0,"id":"fingerbank-collector","managed":0,"pid":"0","status":200}
```
I suppose backend should return `"alive":0`, assigning @jrouzierinverse.

**Comment by jrouzierinverse** [comment]
@nqb Can you verify this patch on Debian?

**Comment by jrouzierinverse** [comment]
@nqb Patch link https://github.com/inverse-inc/packetfence/commit/2a7e707ff1434c7b881d027510768b949238c077.diff

**Comment by nqb** [comment]
Patch applied on a Debian 9 with nightly builds, restart of PF services. I got same behavior.
API response for `fingerbank-collector`, a service that is not running on my setup:
```json
{"alive":1,"enabled":0,"id":"fingerbank-collector","managed":0,"pid":"0","status":200}
```

**Comment by jrouzierinverse** [comment]
Can you run the following command on your install and let me know the output.
```bash
systemctl is-active -q packetfence-fingerbank-collector;echo $?
```

**Comment by jrouzierinverse** [comment]
Also the output of this command
```bash
systemctl show -p SubState,ActiveState packetfence-fingerbank-collector
```

**Comment by nqb** [comment]
```
# systemctl is-active -q packetfence-fingerbank-collector;echo $?
3
# systemctl show -p SubState,ActiveState packetfence-fingerbank-collector
ActiveState=inactive
SubState=dead
```

Exactly the same for `packetfence-haproxy-db`.

**Comment by jrouzierinverse** [comment]
Which API call did you use?
Maybe I am looking at the wrong one.

**Comment by nqb** [comment]
It's a GET on `/api/v1/service/fingerbank-collector/status`

```json
pfperl-api get /api/v1/service/fingerbank-collector/status | python -m json.tool
{
    "alive": 1,
    "enabled": 0,
    "id": "fingerbank-collector",
    "managed": 0,
    "pid": "0",
    "status": 200
}

**Comment by nqb** [comment]
@jrouzierinverse,

Still presents in devel.

**Comment by nqb** [comment]
@jrouzierinverse, any idea why still getting this behavior ?

**Comment by nqb** [comment]
This issue is still presents on Debian 11, not checked on EL8.

**Comment by nqb** [comment]
Output of `systemctl` command is exactly the same (on Debian 11)

**Comment by nqb** [comment]
Not able to replicate on EL8, `alive` equals `0` for a service stopped. `systemctl` commands return exactly same result as for Debian 11.

* EL8: `systemd 239`
* Debian 11: `systemd 247`

**Comment by jrouzierinverse** [comment]
This was fixed by 808d7859afd059ca50c8725c34b825bdfce4966f
